src/components/GameSequence.js

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from 'react'
+import { connect } from 'react-redux'
+import StepSequence from '../components/StepSequence'
+
+export class GameSequence extends PureComponent {
+    constructor(props) {
+        super(props)
+        this.state = {
+            gameSequence: [],
+            nameSequence: ''
+        }
+    }
+    static defaultProps = {
+        classColor: {
+            red: 'sequenceRed',
+            green: 'sequenceGreen',
+            yellow: 'sequenceYellow',
+            blue: 'sequenceBlue',
+        },
+    }
+
+    renderSequence(sequence, index) {
+        return (
+            <StepSequence
+                className={`sequence ${this.props.classColor[sequence]}`}
+                id={`step_${index}`}
```

Comment:
key={`step_${index}`} а если так писать, то уже эта ошибка
warning.js?6327:35 Warning: StepSequence: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)